api/candlepin-api-spec.yaml

```diff
@@ -1215,6 +1215,64 @@ paths:
         default:
           $ref: '#/components/responses/default'
 
+  /crl:
+    get:
+      tags:
+        - Certificate Revocation List
+      description: Retrieves the Certificate Revocation List
+      operationId: getCurrentCrl
+      security: []
+      x-java-response:
+        type: javax.ws.rs.core.Response
+        isContainer: false
+      responses:
+        200:
+          description: Certificate Revocation List successfully retrieved
+          content:
+            text/plain:
+              schema:
+                type: string
+        500:
+          description: CRL file path not defined in config file
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionMessage'
+              example:
+                displayMessage: CRL file path not defined in config file
+                requestUuid: c4347004-8792-41fe-a4d8-fccaa0d3898a
+        default:
+          $ref: '#/components/responses/default'
+
+    delete:
+      tags:
+        - Certificate Revocation List
+      description: Deletes a Certificate from the Revocation List
```

Comment:
The same applies here. Let's avoid capitalization of the display message.

Reply:
Done.